content/en/docs/contributing/release-process.md

```diff
@@ -10,6 +10,61 @@ new release of cert-manager.
 
 ## Prerequisites
 
+<!--
+About "alert" and "pageinfo": https://www.docsy.dev/docs/adding-content/shortcodes/#shortcode-helpers
+-->
+
+{{% pageinfo color="warning" %}}
+
+⛔️ Do not proceed with the release process if you do not satisfy with one of the
+following:
+
+1. The release process **takes about 40 minutes**. You must have time to
+   complete all the steps.
+2. You need to be at Jetstack to create ask for the GitHub and GCP permissions.
+   *(we wish anyone outside Jetstack could be allowed to ask for the
+   permissions; please let us know if that would be of interest to you)*
+3. You need to be a GitHub "owner" on the cert-manager project (if you see the
+   "Settings" tab when looking at the cert-manager project, you **are** an
+   owner). To become a GitHub "owner" of the cert-manager project, [open a
+   PR](https://github.com/jetstack/platform-board/pulls/new) with a link to
+   here.
+4. You need to be added as an "Owner" to the GCP project
+   [cert-manager-release](https://console.cloud.google.com/?project=cert-manager-release).
+   To check if you do have access, try opening [the Cloud Build
+   page](https://console.cloud.google.com/cloud-build?project=cert-manager-release).
+   To get the "Editor" permission on the GCP project, copy-paste the below
+   example template PR into a [new
+   PR](https://github.com/jetstack/platform-board/pulls/new) if you want to get
+   the permissions.
```

Comment:
remove: "if you want to get the permissions"

Reply:
Thanks! Fixed in 2909939 👍